hardware/Netatmo.cpp

```diff
@@ -1363,7 +1383,9 @@ void CNetatmo::GetHomesDataDetails()
 						m_ScheduleNames[index] = schedule["name"].asString();
 						m_ScheduleIDs[index] = schedule_id;
 						m_DeviceHomeID[schedule_id] = homeID;
-						if (!schedule["selected"].empty() && schedule["selected"].asBool())
+						Debug(DEBUG_HARDWARE, "Schedule empty %d", !schedule["selected"].empty());
+						Debug(DEBUG_HARDWARE, "Schedule Bool %d", schedule_selected);
```

Comment:
@JurgenLB , I guess these 2 debug lines can be removed?
And/or add a Debug line within the If or combine all 3 values after the if?